fonts.tex

```diff
@@ -1,4 +1,5 @@
 \setmainfont{STIXTwoText}[
+    Path = fonts/ ,
```

Comment:
Ah I think I tried to also move the `fonts.tex` to the subfolder, maybe that was not clever?